auth/resources/authserver.properties

```diff
@@ -43,11 +43,12 @@ enable_http=true
 enableFaultInjection=false
 perfEnabled=false
 perfLogFile=/var/log/seagate/auth/perf.log
-defaultEndpoint=s3.seagate.com
-s3Endpoints=s3-us-west-2.seagate.com,s3-us.seagate.com,s3-europe.seagate.com,s3-asia.seagate.com
+defaultEndpoint=127.0.0.1
```

Comment:
I guess this will result in UT failure. @sachitanands reported this few days back on simmilar change done by Shalaka. Anything changed since then ?

Reply:
This may have been due to the outdated patch. No changes required.